### ISSUE TITLE
Doesn't work?

### ISSUE THREAD

**Issue by Tanath** [ardagnir/pterosaur#42]
Installed the xpi (easy install). vimbinary points to vim, +clientserver enabled. Tried debugtty but didn't seem to do anything. vim --serverlist shows pterosaur. Running Firefox 36 on Linux with current build of pentadactyl, and I can't seem to get pterosaur working on any input field or text area on any site. Focus always results in insert mode. Attempting to get from insert to normal mode results in passing vim commands to pentadactyl and attempting to use vim movement commands or anything else otherwise just inserts.

**Comment by sschaef** [comment]
Did you add https://github.com/ardagnir/vimbed to your vimrc?

**Comment by Tanath** [comment]
No. Why would I do that? Instructions just say to install the xpi.

**Comment by sschaef** [comment]
I had to add it to my plugins list, otherwise it didn't work. Maybe my plugin manager (Vundle) disables the auto loaded vimbed that comes with pterosaur

**Comment by Tanath** [comment]
I know nothing about vimbed. The vast majority of extensions don't require configuration just to be usable, and the 'easy' instructions imply that shouldn't be the case. I don't use vundle or any other plugin manager.
